.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+DB:
+  - src/**/*.sql
+
+CORE:
+  - src/**/*.cpp
+  - src/**/*.h
+  - '!src/server/scripts/*'
```

Comment:
are you sure it shouldn't be `'!src/server/scripts/**` ? (with double `*`)

Reply:
Yes